.github/workflows/sync-readme.yml

```
@@ -0,0 +1,40 @@
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Sync README to docs
+        run: |
+          cp README.md docs/_README.md
+          echo "✓ Synced README.md to docs/_README.md"
+
+      - name: Check for changes
+        id: changes
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:1:60: Double quote to prevent globbing and word splitting

(shellcheck)